Formula/t/trimal.rb

```diff
@@ -0,0 +1,22 @@
+class Trimal < Formula
+  desc "Automated alignment trimming in large-scale phylogenetic analyses"
+  homepage "https://trimal.readthedocs.io/"
+  url "https://github.com/inab/trimal/archive/refs/tags/v1.5.0.tar.gz"
+  sha256 "3fba2e07bffb7290c34e713a052d0f0ff1ce0792861740a8cec46f40685c6d73"
+  license "GPL-3.0-only"
+  head "https://github.com/inab/trimal.git", branch: "trimAl"
+
+  def install
+    cd "source" do
+      system "make"
+      bin.install "readal", "trimal", "statal"
+    end
+    pkgshare.install "dataset/example.004.AA.fasta"
```

Comment:
if it helps, we can also just download the test file for brew usage rather than shipping it for the bottle.

Reply:
Just pushed an update to use Homebrew's canonical `test.fasta` file.

Reply:
sounds good 👍